google/cloud/storage/benchmarks/storage_throughput_vs_cpu_benchmark.cc

```diff
@@ -162,6 +165,28 @@ int main(int argc, char* argv[]) {
   std::transform(notes.begin(), notes.end(), notes.begin(),
                  [](char c) { return c == '\n' ? ';' : c; });
 
+  auto join_apis = [](std::vector<ApiName> const& apis) {
```

Comment:
JFYI, you could use `absl::StrJoin()` here ([link](https://github.com/abseil/abseil-cpp/blob/master/absl/strings/str_join.h)). Since you already have a `gcs_bm::ToString` function and you're formatting bools not as 0 and 1, you'd want a custom formatter. That would look something like this:

```
struct MyFormatter {
  void operator()(std::string* out, ApiName api) const {
    out->append(gcs_bm::ToString(api));
  }
  void operator()(std::string* out, bool b) const {
    out->append(b ? "true" : "false");
  }
};
...

std::cout << absl::StrJoin(options->enabled_apis, ',', MyFormatter{})
          << absl::StrJoin(options->enabled_crc32, ',', MyFormatter{})
          << absl::StrJoin(options->enabled_md5, ',', MyFormatter{})
```

Your call whether or not that's an improvement... JFYI.